codegen.mts

```diff
@@ -23,11 +23,9 @@ const config: CodegenConfig = {
           'lab',
           'operations',
           'organization',
-          'policy',
           'schema',
         ],
         scalarsOverrides: {
-          DateTime: { type: 'string' },
```

Comment:
By removing `DateTime`, we are relying on Server Preset to wire up `graphql-scalars`'s recommended types, which is `string | Date`
We have to do this now, otherwise there's a mismatch of types in `SchemaPolicy.updatedAt`.